packages/gluestick/src/generator/templates/AppServer.js

```diff
@@ -1,11 +1,23 @@
 /* @flow */
+
 import type { CreateTemplate } from '../../types';
 
 module.exports = (createTemplate: CreateTemplate) => createTemplate`
-  export default {
+/* @flow */
+
+type ApplicationServer = {
+  protocol: string;
+  host: string;
+  port: number;
+  ssetPort: number;
```

Comment:
@zamotany It was already like that so I wasn't paying attention to it. I'll double check because I wonder when did that changed (if before was actually `assetsPort`).